tasks/create-promotional-video-about-the-Labyrinth-game.md

```diff
@@ -0,0 +1,38 @@
+
+
+---
+
+# Labyrinth: Create a Promotional Video about the Labyrinth Game
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+To promote the labyrinth, create a promotional video.
+
+REQUIREMENTS:
+
+- Explore [Video Creating](https://en.wikipedia.org/wiki/List_of_video_editing_software#Free_and_open-source)
+- Include the FOSSASIA logo. You find **FOSSASIA Artwork** on this repository: ***https://github.com/fossasia/fossasia-artwork***
+- Use existing [YouTube videos from the FOSSASIA Channel](https://youtube.com/fossasiaorg) and remix them
```

Comment:
Is this a requirement or a hint and optional?

Reply:
It's a optional step, but will much be appreciated :smile: